lifecycle/lifecycle/django/app/settings.py

```diff
@@ -106,7 +111,7 @@ def is_env_flag_enabled(flag_name: str, default: str) -> bool:
 
 USE_I18N = True
 
-USE_L10N = True
+USE_L10N = False
```

Comment:
This is interesting - why do you disable localization? is it needed for the wildcard changes, or by the way? I see in Django 5 this env is deprecated, but you're still on 4..

Reply:
I changed it by the way when I noticed the datetime format is wrong. ISO format was supposed to be displayed in Django, but got this:
![Screenshot from 2023-03-14 18-29-16](https://user-images.githubusercontent.com/45496492/225089005-c60e83e3-1a3a-4763-98ea-8d84f7c74971.png)
Sorry for putting it here, I think it's too tiny to make an individual PR.